BiometricLocker/BiometricLocker.swift

```diff
@@ -80,16 +99,35 @@ final public class BiometricLocker {
         return false
     }
 
-    public enum When {
-        case now
-        case afterTimeAllowance
-        case afterTimeInterval(TimeInterval)
+    /// Store the NotificationCenter observers for deallocation.
+    private var notificationObservers = [NSObjectProtocol]()
+
+    public init() {
+        self.notificationObservers.append(NotificationCenter.default.addObserver(forName: .UIApplicationDidEnterBackground, object: nil, queue: .main) { _ in
```

Comment:
it would probably be worthwhile to note somewhere in the documentation that we set this up for the consuming app so they don't have to